app/src/org/koreader/launcher/MainActivity.kt

```diff
@@ -697,4 +718,22 @@ class MainActivity : BaseActivity() {
     private fun toMin(milliseconds: Int): Int {
         return if (milliseconds > 0) milliseconds / (1000 * 60) else 0
     }
+
+    private fun showPermissionDialog(message: String, isError: Boolean) {
+        val builder = AlertDialog.Builder(this@MainActivity)
+        val title = resources.getString(if (isError) R.string.error else R.string.warning)
+        builder.setTitle(title).setMessage(message).setPositiveButton("Ok", { _, _ ->
```

Comment:
I think my Android (and us too) says "OK".
```suggestion
        builder.setTitle(title).setMessage(message).setPositiveButton("OK", { _, _ ->
```